examples/sdk_testing.py

```diff
@@ -0,0 +1,67 @@
+from jaqpotpy.jaqpot import Jaqpot
+from jaqpotpy.descriptors.molecular import TopologicalFingerprint
+from jaqpotpy.datasets import JaqpotpyDataset
+from jaqpotpy.api.openapi.models.dataset import Dataset
+from jaqpotpy.api.openapi.models import DatasetType
+
+import pandas as pd
+
+from dotenv import load_dotenv
+
+load_dotenv(".env")
+from jaqpotpy.api.jaqpot_api_client import JaqpotApiClient
+
+jaqpot = JaqpotApiClient()
+# jaqpot.login()
+
+# Get a model by id
```

Comment:
Uncomment all those lines, commented code seems like unfinished business. We use comments either to explain the code or pythondoc, but if the code is commented it looks less presentable. It's OK to have the whole file here just run all the SDK functions and print the result (so uncommented code)